docker-compose.dev.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+  dev_web:
+    build:
+      context: ${PWD}
+      dockerfile: dev.Dockerfile
+    container_name: dev_web
+    environment:
+      - SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock
+      - PAGODA_CACHE_HOSTNAME=dev_cache
+      - PAGODA_DATABASE_HOSTNAME=dev_db
+    volumes:
+    - ${PWD}/:/app
+    - type: bind
+      source: /run/host-services/ssh-auth.sock
+      target: /run/host-services/ssh-auth.sock
+    restart: unless-stopped
+    ports:
+    - "8000:8000"
+  dev_cache:
```

Comment:
Do you need to redefine the cache and db here as opposed to letting the other compose file handle it and this would just be to optionally run the app via _air_?